app/models/container_project.rb

```diff
@@ -9,7 +9,6 @@ class ContainerProject < ApplicationRecord
   has_many :container_replicators
   has_many :container_services
   has_many :containers, :through => :container_groups
-  has_many :containers, :through => :container_groups
```

Comment:
@zeari @moolitayer do we want to split this fix out and get it merged right away?